### PR TITLE
[ADD] pos_invoicing_from_picking

### PR DESCRIPTION
**Pull request by Daniel-CA**
This module allows to invoice from stock picking PoS sale orders.

### CODE REVIEW

File: pos_invoicing_from_picking/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Usage
+=====
+
+To use this module, you need to go to PoS backend sale orders and use the
+picking button to create a picking to be invoiced.
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
```
Comment (elicoidal) [inline pos_invoicing_from_picking/README.rst]: Hi Thanks for your contribution.
Please provide specific URL if any

File: pos_invoicing_from_picking/README.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Credits
```
Comment (elicoidal) [inline pos_invoicing_from_picking/README.rst:27]: I miss the bug tracker section and runbot

File: pos_invoicing_from_picking/oca_dependencies.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+stock-logistics-workflow https://github.com/OCA/stock-logistics-workflow.git
```
Comment (legalsylvain) [inline pos_invoicing_from_picking/oca_dependencies.txt:3]: the url is not necessary, as you use standard version.

**Comment by legalsylvain** [comment]
Hi @Daniel-CA,

It seems that the features of your modules are quiet similar to this module I wrote for V7.
https://github.com/OCA/pos/pull/16/files

Seeing the description, your module realize the (feature 2)[https://github.com/OCA/pos/pull/16/files#diff-bec4f0b12d68b31529c66ae8f7dbd87bR29]. (invoice draft order).
- What do you think about merging features / module / tests ? (for V8)
- What about to simply call your module 'pos_invoicing' ?

I can PR against your PR, if you're interested.

Kind regards.

**Comment by Daniel-CA** [comment]
Hi @legalsylvain, 
Is fine for me, I didn't notice that there was something similar for V7.
So go ahead with the PR for merging features, about the name change I agree too. :+1: 

Regards

**Pull request closed without merging**